### ISSUE TITLE
Rel  3.4 Installation Docu - Tomcat config - GC

### ISSUE THREAD

**Issue by stefanCCS** [kitodo/kitodo-production#4841]
In the Release 3.4 Installation documentation 
https://github.com/kitodo/kitodo-production/wiki/Installationsanleitung-f%C3%BCr-Kitodo.Production-3.4
there is configuration option for tomcat9 mentioned:
`JAVA_OPTS="-Djava.awt.headless=true -XX:+UseConcMarkSweepGC -Xmx4G -XX:MaxPermSize=256m"
`
Question is, if really "UseConcMarkSweepGC " should be used, or the for Java 11/Tomcat9 default GC "UseG1GC".

Please clarify.

**Comment by markusweigelt** [comment]
Yes, "-XX:+UseConcMarkSweepGC" is deprecated and should be changed to "-XX:+UseG1GC" http://openjdk.java.net/jeps/291

**Comment by stefanCCS** [comment]
@solth : This is still open, and IMO should be changed/added in installation documentation for Release 3.5
(see also #4842)

**Comment by solth** [comment]
Updated [Kitodo 3.x installation guide](https://github.com/kitodo/kitodo-production/wiki/Installationsanleitung-f%C3%BCr-Kitodo.Production-3.X) as suggested.
